src/Integration/LocalServices/IUnboundSolutionChecker.cs

```diff
@@ -0,0 +1,35 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+namespace SonarLint.VisualStudio.Integration
+{
+    internal interface IUnboundSolutionChecker : IBindingChecker
```

Comment:
Decided to reuse the interface for no particular reason other than it's exactly the method I wanted to add 🤷‍♀️

Reply:
Hhmm, that's confusing. It might be taken to suggest that the `ErrorListInfoBarController` could be validly be handed an `UnboundSolutionChecker`, which isn't really what we want.

I think it would be simpler to keep the interfaces completely separate for now, so that the compiler won't let us pass the "wrong" type to the `ErrorListInfoBarController`.

Having identical interfaces seems like a design smell that perhaps the responsibilities aren't quite right yet.

Reply:
Good point 👍